File: autoimageslider/src/main/java/com/codebyashish/autoimageslider/Interfaces/ItemsListener.java
package com.codebyashish.autoimageslider.Interfaces;

import com.codebyashish.autoimageslider.Enums.ImageActionTypes;

public interface ItemsListener {
    void onItemChanged (int position);
    void onTouched (ImageActionTypes actionTypes, int position);
    void onItemClicked (int position);
}

File: autoimageslider/src/main/java/com/codebyashish/autoimageslider/Interfaces/ItemsListener.kt
package com.codebyashish.autoimageslider.Interfaces

import com.codebyashish.autoimageslider.Enums.ImageActionTypes

interface ItemsListener {
    fun onItemChanged(position: Int)
    fun onTouched(actionTypes: ImageActionTypes?, position: Int)
    fun onItemClicked(position: Int)
}
